ClientPlugin/Plugin.cs

```diff
@@ -98,7 +100,8 @@ private void CustomUpdate()
         // ReSharper disable once UnusedMember.Global
         public void OpenConfigDialog()
         {
-            MyGuiSandbox.AddScreen(new PluginConfigDialog());
+            Instance.ConfigGenerator.SetLayout<Simple>();
```

Comment:
See my question at `internal class None`

Reply:
See above answer.
It is worth noting None is not a null layout, it is a layout in and of itself.